libs/components/tabs/src/lib/modules/vertical-tabset/vertical-tabset.component.scss

```diff
@@ -38,28 +66,42 @@
 }
 
 .sky-vertical-tabset-show-tabs-btn {
-  margin-top: $sky-margin;
+  margin-top: var(--sky-override-vertical-tabset-show-tabs-margin-top, 0);
 }
 
 .sky-vertical-tabset-group-container {
   flex-basis: 25%;
   overflow-y: auto;
+  padding: var(
+    --sky-override-vertical-tabset-group-container-padding,
+    var(--sky-space-inset-vertical-top-m) 0
+      var(--sky-space-inset-vertical-bottom-m)
+  );
 }
 
 :host-context(sky-modal-content) > :host .sky-vertical-tabset {
-  margin: -15px;
+  margin: var(
+    --sky-override-vertical-tabset-modal-margin-adjustment,
```

Comment:
We adjust the positioning of vertical tabs within modals. The existing `-15px` was due to that being default theme's margin on modal content. We did not change this for modern, but are tokenizing here so that the text of the tabs will align with the modal header text and footer content.